modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -75,6 +75,10 @@ ComputeMultipleInelasticStress::ComputeMultipleInelasticStress(const InputParame
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_base_name + "elasticity_tensor")),
     _elastic_strain_old(getMaterialPropertyOld<RankTwoTensor>(_base_name + "elastic_strain")),
     _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment")),
+    _has_compliance(hasMaterialProperty<RankFourTensor>(_base_name + "compliance_tensor")),
```

Comment:
Yes!  This really bugged me, but i don't know how to get around it.   While writing this PR i was pretty unconvinced of its usefulness and robustness and unless this issue can be resolved in a simple way, the PR should probably be terminated.